### PR TITLE
moving ChannelROI to sbnobj

### PR DESCRIPTION
**Pull request by jzennamo**
Moving ChannelROI to SBNObj lets SBND use it and allows for a common release of TITUS

### CODE REVIEW

**Comment by mmrosenberg** [comment]
trigger build LArSoft/lar*@LARSOFT_SUITE_v09_75_03 SBNSoftware/sbnobj#92

**Comment by mmrosenberg** [comment]
trigger build LArSoft/lar*@LARSOFT_SUITE_v09_75_03 SBNSoftware/sbnobj#92

**Review by SFBayLaser — APPROVED** [review]
Yes, I am happy with moving this  data object to SBN. I suppose one day it should go to LArSoft but for now this makes it usable for SBND and ICARUS. Just reading the changes I think everything is covered here.

File: icaruscode/IcarusObj/classes.h [hunk 1]
```diff
@@ -6,7 +6,7 @@
 #include "icaruscode/IcarusObj/CRTTPCMatchingInfo.h"
 #include "icaruscode/IcarusObj/OpDetWaveformMeta.h"
 #include "icaruscode/IcarusObj/PMTWaveformTimeCorrection.h"
-#include "icaruscode/IcarusObj/ChannelROI.h"
+//#include "icaruscode/IcarusObj/ChannelROI.h"
```
Comment (PetrilloAtWork) [inline icaruscode/IcarusObj/classes.h:9]: Or you can just remove it:
```suggestion
```

**Review by PetrilloAtWork — APPROVED** [review]
This side looks good to me.

**Comment by mmrosenberg** [comment]
trigger build LArSoft/lar*@LARSOFT_SUITE_v09_75_03 SBNSoftware/sbnobj#92

**Comment by mmrosenberg** [comment]
trigger build LArSoft/lar*@LARSOFT_SUITE_v09_75_03 SBNSoftware/sbnobj#92

**Comment by mmrosenberg** [comment]
trigger build

**Pull request merged**